unlock_reference.go

```diff
@@ -35,3 +35,7 @@ func (r *ReferenceUnlock) Type() UnlockType {
 func (r *ReferenceUnlock) Size() int {
 	return ReferenceUnlockSize
 }
+
+func (r *ReferenceUnlock) WorkScore(workScoreStructure *WorkScoreStructure) WorkScore {
+	return workScoreStructure.WorkScoreEd25519Signature
```

Comment:
Btw, we never charge for these right? Because we never analyse inputs anyway? Maybe we should switch to charge costs if someone creates these things on the output side.

Reply:
changed now to just be data.